### DISCUSSION TITLE
I changed the FontSize, but it did not work

### DISCUSSION THREAD

**Discussion by ZifanLee** [vercel/hyper#6373] [Q&A]
macOS Big Sur
the font size is quite small, I changed the fontsize parameter in config file, but it did not work. 
when I changed other parameter, it worked normally.
<img width="842" alt="image" src="https://user-images.githubusercontent.com/82391615/157427519-4411649e-7b35-471c-ba68-c190f7172f7d.png">

**Comment by LabhanshAgrawal** [comment]
What are you setting it to?

**Reply by WS-Na** [reply]
this is my setup.
fontFamily: '"Recursive Mono Casual Static","3270Medium Nerd Font"',

**Reply by LabhanshAgrawal** [reply]
I meant the size

**Reply by WS-Na** [reply]
in ~/.hyper.js have this setting
module.exports = {
    config: {
        // choose either 'stable' for receiving highly polished,
        // or 'canary' for less polished but more frequent updates
        updateChannel: 'stable',
        // default font size in pixels for all tabs
        `fontSize: 31,`
